linux_os/guide/system/apparmor/all_apparmor_profiles_in_enforce_complain_mode/ansible/shared.yml

```diff
@@ -0,0 +1,27 @@
+# platform = multi_platform_sle
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+{{{ ansible_instantiate_variables("var_apparmor_mode") }}}
+
+- name: Ensure all AppArmor Profiles are reloaded
```

Comment:
Could you include the `{{{ rule_title }}}` prefix on tasks names, please?
https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible